simulations/notebooks/Sampling-subclones.ipynb

```diff
@@ -15,10 +15,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "id": "e1ba35b1",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Global seed set to 0\n"
+     ]
+    },
+    {
+     "ename": "AnnDataReadError",
```

Comment:
Would cleaning the outputs be feasible in this case? (If the notebook doesn't take enormous time to run, it's a good practice to remove the outputs, as the diffs will be minimized and one doesn't leak the derived data accidentally).

Reply:
Yes sorry I just forgot but you are entirely right.